ml_model/labelbox_api.py

```diff
@@ -0,0 +1,169 @@
+""" File: labelbox_api.py
+
+Description: Functions for using the labelbox API and creating yolo model segmentation labels. 
+Using these functions will require either creating a CLI application or creating a script. 
+"""
+
+import labelbox as lb
+import labelbox.types as lb_types
+from glob import glob
+import os
+import uuid
+import cv2
+import numpy as np
+import ndjson
+import urllib.request
+from PIL import Image
+
+# Add the needed API key
+API_key = ""
```

Comment:
we can use local .env files to manage API keys (https://jonathansoma.com/lede/foundations-2019/classes/apis/keeping-api-keys-secret/)